src/components/panel/panel.tsx

```diff
@@ -251,8 +275,8 @@ export class Panel implements ConditionalSlotComponent, InteractiveComponent {
   };
 
   dismiss = (): void => {
-    this.dismissed = true;
     this.closed = true;
+    this.closable = false;
```

Comment:
why is closable being set here? closable is independent of closing or dismissing right?

Reply:
yup, closable is independent of dismissing.